### PR TITLE
rac-gcp-deploy improvements

### PR DESCRIPTION
**Pull request by rmorais**
This pr introduces several improvements, mainly to make our life easier while publishing libraries:
- new job to release libraries
- possibility to pass options to sbt commands
- cache key is now a parameter

### CODE REVIEW

**Review by lkrystkowiak-ovoenergy — APPROVED** [review]
lgtm 👍🏼

File: rac-gcp-deploy/orb.yml [hunk 1]
```diff
@@ -197,22 +231,16 @@ jobs:
           api-key: <<parameters.artifactory-api-key>>
       - run:
           name: "Compile"
-          command: sbt update compile test:compile evicted
+          command: sbt -v +update +compile +test:compile +evicted <<parameters.sbt-extra-options>>
```
Comment (jensraaby) [inline rac-gcp-deploy/orb.yml:234]: Nice to have the extra options 👍

File: rac-gcp-deploy/orb.yml [hunk 2]
```diff
@@ -226,19 +254,27 @@ jobs:
...
         default: ~/project
+      cache-key:
+        type: string
+        default: *cache-key
       cache-suffix:
         description: "Suffix to use for the cache key, e.g. -v2, to be able to clear the cache"
         default: ""
         type: string
+      sbt-extra-options:
+        description: "Extra options being passed to sbt, like '--mem 2048'"
```
Comment (jensraaby) [inline rac-gcp-deploy/orb.yml:265]: Is it a mistake that there are two `-` here?

Comment (rmorais) [reply rac-gcp-deploy/orb.yml:265]: yes it is

**Review by jensraaby — APPROVED** [review]
🥳  this looks great - good learning exercise for me

**Pull request merged**